docs/app/Examples/modules/Accordion/Types/AccordionExamplePanelsProp.js

```diff
@@ -1,10 +1,22 @@
-import React from 'react'
-import { Accordion } from 'semantic-ui-react'
+import React, { PropTypes } from 'react'
+import { Accordion, Icon } from 'semantic-ui-react'
 import faker from 'faker'
 import _ from 'lodash'
 
-const panels = _.times(3, () => ({
-  title: faker.lorem.sentence(),
+const CustomTitle = ({ text }) => (
```

Comment:
Done, let me now if there should be any description or any other title (writing good documentation is really really hard :smile:)